<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_translations">
       <attribute name="title">
        <string>Translations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="h_layout_lang_trans" stretch="1,2,1,8">
          <item>
           <widget class="QLabel" name="lb_lang_sel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_lang_sel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_trans_sel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Translation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_trans_sel"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lb_bitmap_show_translations">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_fonts_only">
       <attribute name="title">
        <string>Fonts Only</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="lb_user_input">
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_input_string"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lb_utf8_display">
            <property name="text">
             <string>UTF-8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_unicode_display">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lb_bitmap_show_fonts_only">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1,0,1,0,1">
      <item>
       <widget class="QLabel" name="lb_bitmap_border">
        <property name="text">
         <string>Bitmap Border</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_bitmap_border_top">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_bitmap_border_top"/>
      </item>
      <item>
       <widget class="QLabel" name="lb_bitmap_border_bottom">
        <property name="text">
         <string>Bottom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_bitmap_border_bottom"/>
      </item>
      <item>
       <widget class="QLabel" name="lb_bitmap_border_left">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_bitmap_border_left"/>
      </item>
      <item>
       <widget class="QLabel" name="lb_bitmap_border_right">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_bitmap_border_right"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
